be/src/storage/lake/tablet_manager.cpp

```diff
@@ -108,12 +113,17 @@ std::string TabletManager::tablet_metadata_lock_location(int64_t tablet_id, int6
     return _location_provider->tablet_metadata_lock_location(tablet_id, version, expire_time);
 }
 
+std::string TabletManager::global_schema_cache_key(int64_t schema_id) {
+    return fmt::format("A{}", schema_id);
```

Comment:
`ABC` might be too hack, how about
* global_schema -> GS
* tablet_schema -> TS
* tablet_latest_ma -> TL

Reply:
updated